packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -752,6 +752,24 @@ export const formatters: Formatters = {
     desc: 'Database Deleted',
     format: ({ user, name }) => `User [${user}] deleted database [${name}]`,
   },
+  [eventCodes.APPLICATION_CREATED]: {
+    type: 'app.create',
```

Comment:
Here we have `app.create` but in the fixture above we have `app.created`. What's the difference?

Reply:
Whoops. That would be a typo from me, good catch.

Reply:
This is fixed, but I'm sort of confused why it worked -- is the `type` not used for anything outside of the details page?